semperfiwebdesign/all-in-one-seo-pack
# Improved display of date picker in Safari

Issue #2739

## Proposed changes
The jQuery UI date picker doesn't look good in Safari so this adds CSS to improve the look.

## Types of changes
- Improves existing functionality
- Improves existing code

## Checklist

## Testing instructions
- Go to All in One SEO > XML Sitemap > Additional Pages and check the styling of the date picker in the Last Modified option
- Verify the styling in all modern browsers (specifically IE and Safari)
- Perform the same tests with the XML Sitemap and Video Sitemap in Pro

## Further comments
I've tested this in mobile Safari and it does not affect the look or behavior of the date field.  You should verify this does not affect mobile browsers on Android devices.
